apps/rock-paper-scissors/src/pages/home/details/Details.tsx

```diff
@@ -0,0 +1,71 @@
+import { Hex } from '@gear-js/api';
+import { BackButton, Detail } from 'components';
+import { SVGType } from 'types';
+import styles from './Details.module.scss';
+
+type Props = {
+  heading: string;
+  game: string;
+  round: string;
+  contract: Hex | undefined;
+  bet: string | undefined;
+  entry: string | undefined;
+  move: string | undefined;
+  reveal: string | undefined;
+  SVG: SVGType;
+  admin?: boolean;
+  players?: Array<Hex> | Array<string> | [];
+  onBackClick: (arg: string) => void;
+};
+
+
+function Details({
+  onBackClick,
+  heading,
+  game,
+  round,
+  contract,
+  players,
+  bet,
+  entry,
+  move,
+  reveal,
+  SVG,
+  admin
+}: Props) {
+  const handleRouteChange = () => {
+    if (admin) { onBackClick('lobby admin') } else { onBackClick('game') }
+  }
```

Comment:
It should be prop, no need for logic in a dumb component

Reply:
fixed